linux_os/guide/system/software/integrity/software-integrity/aide/aide_disable_silentreports/ansible/ubuntu.yml

```diff
@@ -0,0 +1,19 @@
+# platform = multi_platform_ubuntu
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: Ensure aide is installed
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - aide
+
```

Comment:
```suggestion
```

Reply:
There is one extra blank line here.

Reply:
thank you for the feedback. PR has been rebased with removing the ansible remediations in favor of using templates